### PR TITLE
tv-actions tests completed

### PR DESCRIPTION
**Pull request by alisherk**

### CODE REVIEW

**Comment by alisherk** [comment]
@trybick do you know why this build is failing?

**Comment by trybick** [comment]
@alisherk Yeah looks like there are prettier warnings. When you commit there should be a pre-commit hook that runs to run the formatting command automatically. Do you see that in your terminal when you commit? 

You can try running `cd front` and then `yarn lint` or `npm run lint` and that should fix those errors.

I'm a little confused tho. Don't we already have tests for getColorForShowIds or handleErrors?

**Comment by alisherk** [comment]
@trybick sorry about that there were duplicates in my local project.

File: front/src/store/tv/actions.ts [hunk 1]
```diff
@@ -85,8 +85,6 @@ export const requestBasicShowInfoAction = (): AppThunk => async (dispatch, getSt
     validCachedIds.forEach(id => {
       combinedData[id] = cachedBasicShowInfo[id];
     });
-
-  // Fetch data for ids that are not cached and add to combinedData
```
Comment (trybick) [inline front/src/store/tv/actions.ts]: why was this deleted?

**Review by trybick — COMMENTED** [review]
Hey @alisherk thanks for doing this :) I will re-open an Issue to track more tests.

If you get around to writing more tests, could we test the UI? Like test components with snapshots or verifying certain elements exist.

File: front/src/store/tv/actions.ts (hunk 1 above)
Comment (alisherk) [reply front/src/store/tv/actions.ts]: sorry I did it by mistake. I was trying to understand how this action works and may have touched the comment. The comment should still be there tho. Regarding testing components: yep we can start testing there as well. Everything should be tested.

**Comment by trybick** [comment]
@alisherk If you look at the Files tab of this PR you can see the Readme and `front/src/store/tv/actions.ts` are still being modified. Before merging, can you revert these?

You could simply delete or add back to revert it, or run a command like this on each file
`git checkout origin/master -- front/src/store/tv/actions.ts`

**Comment by alisherk** [comment]
Did it do the trick?

**Comment by trybick** [comment]
yep thanks! @alisherk   do you want to do component snapshots next?
